site2/docs/deploy-bare-metal.md

```diff
@@ -44,28 +43,25 @@ To run Pulsar on bare metal, the following configuration is recommended:
 
 > **Note**
 >
-> Broker is only supported on 64-bit JVM.
-
-> If you do not have enough machines, or to try out Pulsar in cluster mode (and expand the cluster later),
+> * Broker is only supported on 64-bit JVM.
+> * If you do not have enough machines, or to try out Pulsar in cluster mode (and expand the cluster later),
 > you can deploy a full Pulsar configuration on one node, where Zookeeper, the bookie and broker are run on the same machine.
-
-> If you do not have a DNS server, you can use the multi-host format in the service URL instead.
-
+> * If you do not have a DNS server, you can use the multi-host format in the service URL instead.
 Each machine in your cluster needs to have [Java 8](https://adoptopenjdk.net/?variant=openjdk8) or [Java 11](https://adoptopenjdk.net/?variant=openjdk11) installed.
 
 The following is a diagram showing the basic setup:
 
 ![alt-text](assets/pulsar-basic-setup.png)
 
-In this diagram, connecting clients need to be able to communicate with the Pulsar cluster using a single URL. In this case, `pulsar-cluster.acme.com` abstracts over all of the message-handling brokers. Pulsar message brokers run on machines alongside BookKeeper bookies; brokers and bookies, in turn, rely on ZooKeeper.
+In this diagram, connecting clients need to communicate with the Pulsar cluster using a single URL. In this case, `pulsar-cluster.acme.com` abstracts over all of the message-handling brokers. Pulsar message brokers run on machines alongside BookKeeper bookies; brokers and bookies, in turn, rely on ZooKeeper.
```

Comment:
```suggestion
As shown in this diagram, connecting clients communicate with the Pulsar cluster using a single URL. More precisely, the URL `pulsar-cluster.acme.com` abstracts over all message-handling brokers. Pulsar message brokers run on machines alongside BookKeeper bookies; brokers and bookies, in turn, rely on ZooKeeper.
```

I am not sure about the meaning of "abstracts over all of the message-handling brokers"